scripts/build/download_libbpf_headers.sh

```diff
@@ -44,4 +44,13 @@ if [ "$?" -ne 0 ]; then
     exit 1
 fi
 
+# Fetch compact vmlinux file from cilium's ebpf repository.
+# This is not a libbpf header per-se, but it's close enough that we put it in the same location.
+curl -s -o "$HEADERS_DIRECTORY"/vmlinux.h \
+    https://raw.githubusercontent.com/cilium/ebpf/v${CILIUM_VMLINUX_VERSION}/examples/headers/common.h
```

Comment:
Consider updating the cilium_vmlinux_version to v0.10.0 - latest release

Reply:
updated